tests/chart_tests/test_astronomer_houston_task_metrics.py

```diff
@@ -38,6 +36,37 @@ def test_astronomer_cleanup_task_usage_cron_feature_enabled(self, kube_version):
         )
         assert docs[0]["spec"]["schedule"] == "40 23 * * *"
 
+    def test_astronomer_populate_hourly_task_audit_metrics_cron_defaults(
+        self, kube_version
+    ):
+        docs = render_chart(
+            kube_version=kube_version,
+            values={"global": {"taskUsageMetricsEnabled": False}},
+            show_only=[
+                "charts/astronomer/templates/houston/cronjobs/houston-populate-hourly-task-audit-metrics.yaml",
+            ],
+        )
+        assert len(docs) == 0
+
+    def test_astronomer_populate_hourly_task_audit_metrics_cron_feature_enabled(
+        self, kube_version
+    ):
+        docs = render_chart(
+            kube_version=kube_version,
+            values={"global": {"taskUsageMetricsEnabled": True}},
+            show_only=[
+                "charts/astronomer/templates/houston/cronjobs/houston-populate-hourly-task-audit-metrics.yaml",
+            ],
+        )
+
+        assert len(docs) == 1
+        assert docs[0]["kind"] == "CronJob"
+        assert (
+            docs[0]["metadata"]["name"]
+            == "release-name-houston-populate-hourly-task-audit-metrics"
+        )
+        assert docs[0]["spec"]["schedule"] == "57 * * * *"
```

Comment:
We should have a test to verify that custom schedules work.

Reply:
done